frontend/tests/functional/detailed/settings/general.test.ts

```
@@ -0,0 +1,198 @@
+	let page: Page;
+	test.beforeAll(async ({ browser }) => {
+		// Create a unique page to use for all the tests on this user group and login
+		page = await browser.newPage();
+		const loginPage = new LoginPage(page);
+		await loginPage.goto();
+		await loginPage.login(LoginPage.defaultEmail, LoginPage.defaultPassword);
+		await expect(page).toHaveURL('/analytics');
+	});
+
+	test.use({
+		page: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)